src/components/grapher/Grapher.vue

```diff
@@ -114,16 +137,8 @@ export default Vue.extend({
       return this.$store.getters.getBackHashOffsetY + this.$store.getters.getFrontHashOffsetY;
     },
     fourStepGridOffsetsX(): number[] {
-      // Do not render a vertical line if there is a yard line
```

Comment:
This makes the 4step grid look weird without yardlines so I removed it.

Reply:
Can you look into possibly creating two groups of grid lines:

1. Vertical grid lines that are not on a yard line
2. Vertical grid lines that are on a yard line

Then, do a `v-if`, `v-else`, and `v-else-if` ([v-else docs](https://vuejs.org/v2/guide/conditional.html#v-else)) on `yardlines`.

- If `yardlines === true`, show the yard lines
- Else if `fourStepGrid === true`, show the grid lines in place of the yard lines
- Else, do not show anything

This way, we are not generating 10 extra grid lines while `yardlines` is true.